bundle/manifests/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -221,6 +224,11 @@ spec:
                       items:
                         type: string
                       type: array
+                    labels:
```

Comment:
Looks like this crossed over from the message labeling PR?

Reply:
Actually this is not intended to be part of this PR, but until the bundle generation is bound to CRD generation, we need to update the bundle with `make generate-bundle` manually.